public/apps/configuration/app-router.tsx

```diff
@@ -147,6 +150,15 @@ export function AppRouter(props: AppDependencies) {
         </Route>
         <EuiPageBody>
           <Switch>
+            <Route
+              path={buildUrl(ResourceType.roles, Action.view, ':roleName', SubAction.mapuser)}
+              render={(match) => (
+                <RoleView
```

Comment:
Open for discussion. I think editing mapped user is better to have its own component similar to RoleEdit, say RoleEditMappedUser. @yuxitang-amzn

Reply:
This sounds reasonable. Though on the UI it is attached to role view page, in the coding part we could still name it with 'edit' which looks making more sense.

Reply:
Agreed. Moved map user part to it's own component named `RoleEditMappedUser`